test/liquidation/liquidation.test.js

```diff
@@ -2,7 +2,7 @@ const { expect } = require("chai")
 const { to1e18 } = require("../helpers/contract-test-helpers")
```

Comment:
This file is no longer only a liquidation happy path. Also, in #53, we introduced system tests under `test/system/liquidation.test.js`.

Given that we have more tests than just the liquidation happy path in both cases, I propose we do the rename to keep them as:
```
test/integration/integration.test.js
test/system/system.test.js
```
for now and if we define enough test cases that it will make sense, split those files into several smaller ones.

Reply:
Done in 2c29566b2fba6d7c26e266b118d46d24f18e4d5c.